command/certificate/extractPE.go

```diff
@@ -0,0 +1,168 @@
+package certificate
+
+import (
+	"debug/pe"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"go.mozilla.org/pkcs7"
+	"io"
+	"os"
+	"path"
+)
+
+var ErrCodeSignCertsNoCertificateTableFound = errors.New("No Certificate Table found")
+
+func extractPECommand() cli.Command {
+	return cli.Command{
+		Name:        "extract-pe",
+		Action:      cli.ActionFunc(extractPEAction),
+		Usage:       `TODO`,
+		UsageText:   `TODO`,
+		Description: `TODO`,
```

Comment:
Please add docs for here. We use a slightly modified version of markdown for this, you can see how other docs are done.